ext/bin/pdbbox-init

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-set -ueo pipefail
+set -uexo pipefail
```

Comment:
Maybe set -x after we do the pg password setup? Might not matter because we just use this in testing, but something I wondered about.

Reply:
Ooooops I did not mean to leave the -x in there.